prost-build/src/ident.rs

```diff
@@ -26,17 +26,27 @@ pub fn to_snake(s: &str) -> String {
         "self" | "super" | "extern" | "crate" => ident += "_",
         _ => (),
     }
+
+    if !ident.chars().nth(0).unwrap_or('x').is_alphabetic() {
```

Comment:
Also, please add a comment here to this effect (I went down a rabbit hole of thinking about unicode idents, particularly since they just became available in Rust, before realizing it's not necessary for this code to handle it):

```
// Add an underscore ('_') prefix to identifiers which are valid by the [Protobuf rules][1], but which are not valid
// Rust identifiers. In particular, Protobuf allows identifiers to begin with numerals, whereas Rust does not. The
// reverse -- identifiers which are valid in Rust (such as non-ASCII `XID_Start` characters) are never produced
// by `protoc`, and thus do not need to be handled.
// [1]: https://developers.google.com/protocol-buffers/docs/reference/proto3-spec
```